File: task3.kt
import java.util.Scanner;
import java.util.InputMismatchException
fun main() {
    val scanner = Scanner(System.`in`)
    val numbers = scanner.nextInt()
    if (numbers <= 0) {
        println("That is impossible.")
        return
    }
    var count = 0
    for (i in 1..numbers) {
        val perem = scanner.nextInt()
        var ost = perem
        if (ost < 0) {
            ost = -ost
        }
        while (ost > 0) {
            val digit = ost % 10
            if (digit % 3 == 0) {
                count += digit
            }
            ost /= 10
        }
    }
    println("Answer: ")
    println(count)
}

File: Task3/task3.rb
numbers = gets.to_i
count = 0
while numbers > 0
  perem = gets.to_i
  while perem != 0
    ost = perem % 10
    if ost % 3 == 0
      count += ost
    end
    perem /= 10 
  end
  numbers -= 1
end
puts count
